spring-cloud-gcp-vision/src/main/java/org/springframework/cloud/gcp/vision/CloudVisionTemplate.java

```diff
@@ -50,6 +51,11 @@ public CloudVisionTemplate(ImageAnnotatorClient imageAnnotatorClient) {
 		this.imageAnnotatorClient = imageAnnotatorClient;
 	}
 
+	public String extractTextFromImage(Resource resource) throws IOException {
```

Comment:
What's the source of the `IOException`? We probably need to wrap it into a custom runtime exception.
I believe the templates in Spring generally try to avoid checked exceptions. @artembilan WDYT?

Reply:
Well, you can simply wrap it into the `UncheckedIOException` 😄

Reply:
Maybe a `CloudVisionException` wrapping `IOException`?
This also reminds me that we wanted to check for error responses from Cloud Vision and wrap them into exceptions.